src/v2/Apps/Authentication/Server/checkForRedirect.ts

```diff
@@ -14,6 +15,8 @@ export const checkForRedirect = ({ req, res }) => {
   let newRedirect
   if (redirectTo === ("/reset_password" || "/user/delete")) {
     newRedirect = "/"
+  } else if (!!redirectTo) {
+    newRedirect = sanitizeRedirect(redirectTo)
   } else {
     newRedirect = redirectTo
   }
```

Comment:
can we leave something like this?

```javascript
if (redirectTo === ("/reset_password" || "/user/delete")) {
    newRedirect = "/"
  } else if (!!redirectTo) {
    newRedirect = sanitizeRedirect(redirectTo)
  }
```

Reply:
Absolutely! Updated in [this](https://github.com/artsy/force/pull/9817/commits/8d7707824dbc7f144ca6df4431aeaaf5fa3a94b6) commit

Reply:
Nice. Thank you!